src/grove/settings.py

```diff
@@ -37,6 +37,7 @@
 
 INSTALLED_APPS = [
     'vocabs',
+    'grove',
```

Comment:
If you move the management commands to the vocabs app, then you can remove this line.